app/Providers/RouteServiceProvider.php

```diff
@@ -510,6 +510,9 @@ private function mapApiRoutes()
                 Route::get('news', 'NewsController@index');
                 Route::get('status', 'StatusController@status');
                 Route::get('version', 'StatusController@status');
+
+                Route::get('airports/find', 'AirportController@find');
```

Comment:
call it `/search` and also implement the same thing used for the flights search - that uses a build-in search functionality of the repository interface. That way they can both be leveraged and used in the same way